components/gitpod-protocol/src/team-subscription-protocol.ts

```diff
@@ -18,6 +18,8 @@ export interface TeamSubscription {
     paymentReference: string;
     cancellationDate?: string;
     deleted?: boolean;
+    /** If this flag is set slots are not eligibile for clusters with "more-resources" - even if their plan might be */
+    excludeFromMoreResources?: boolean;
```

Comment:
Nit: Can't be `undefined`.

```suggestion
    excludeFromMoreResources: boolean;
```

Reply:
Auto-save did not auto-save :rolling_eyes: